### ISSUE TITLE
너무 좋은 튜토리얼 감사합니다! 오타인거 같아서 남겨봅니다.

### ISSUE THREAD

**Issue by merovingiaan** [golbin/TensorFlow-Tutorials#11]
텐서플로우 공부하면서 많은 도움 받았습니다. 감사합니다.
cnn layer 쪽 보다가 오타같아 보여서 적습니다.

02 - tf.layers.py 에 32번 라인에 L2가 L3 이 아닌가 싶습니다.
=> **L2** = tf.layers.dropout(**L2**, 0.5, is_training)

**Comment by golbin** [comment]
아! 맞습니다. 곧 수정하겠습니다. 감사합니다!
